docroot/modules/custom/va_gov_build_trigger/src/Plugin/Block/ContentReleaseStatusBlock.php

```diff
@@ -0,0 +1,262 @@
+
+    return $row;
+  }
+
+  /**
+   * Add log links to table.
+   *
+   * @param array $table
+   *   Drupal table render array.
+   *
+   * @return array
+   *   Drupal table render array.
+   */
+  private function addLogLinks(array $table) : array {
+    if (
+      $this->environmentDiscovery->isTugboat() &&
+      $tugboat_environment_id = $this->environmentDiscovery->getTugboatBuildEnvironmentId()
+    ) {
+      $log_url = Url::fromUri(
+        "https://tugboat.vfs.va.gov/log/{$tugboat_environment_id}",
+        [
+          'attributes' => [
+            '_target' => 'blank',
+          ],
+        ]
+      );
+      $log_link = $this->linkGenerator->generate(
+        $this->t('View logs'),
+        $log_url
+      );
+
+      $table['#header'][] = $this->t('Logs');
+      $table['#rows'][] = $log_url;
+    }
+
+    return $table;
+  }
+
+  /**
+   * Get the status icon for an advancedqueue job state.
+   *
+   * @param string $state
+   *   Advancedqueue job state.
+   *
+   * @return array
+   *   Table cell array with job status icon.
+   */
+  private function getStatusIcon(string $state) : array {
+    $class = '';
+    $icon = '';
+
+    switch ($state) {
+      case 'queued':
+        $icon = '🕐';
+        break;
+
+      case 'processing':
+        $class = 'status-animated';
+        $icon = '🔨';
+        break;
+
+      case 'success':
+        $icon = '✅';
+        break;
+
+      case 'failure':
+        $icon = '❌';
+        break;
+
+      default:
+        $icon = '❓';
+        break;
+    }
+
+    return [
+      'data' => $icon,
+      'class' => $class,
+    ];
+  }
+
+  /**
+   * Get the human readable status for an advancedqueue job state.
+   *
+   * @param string $state
+   *   Advancedqueue job state.
+   *
+   * @return array
+   *   Table cell array with human-readable job status.
+   */
+  private function getHumanReadableStatus(string $state) : array {
+    $status = '';
+    switch ($state) {
+      case 'queued':
+        $status = $this->t('Pending');
+        break;
+
+      case 'processing':
+        $status = $this->t('In Progress');
+        break;
+
+      case 'success':
+        $status = $this->t('Success');
+        break;
+
+      case 'failure':
+        $status = $this->t('Error');
+        break;
+
+      default:
+        $status = $this->t('Unknown');
+        break;
+    }
+
+    return [
+      'data' => $status,
+      'class' => "status-{$state} ajax-progress-throbber",
+    ];
+  }
+
+  /**
+   * Get content release build job.
+   *
+   * @return object|bool
+   *   The job's database row object, or FALSE if none is found.
+   */
+  private function getCommandRunnerJob() {
+    return $this->database->select('advancedqueue', 'aq')
+      ->condition('aq.queue_id', 'command_runner')
+      ->fields('aq')
+      ->range(0, 1)
+      ->orderBy('available', 'DESC')
+      ->execute()
+      ->fetchObject();
+  }
```

Comment:
```suggestion
  private function getCommandRunnerJob() : ?stdClass {
    return $this->database->select('advancedqueue', 'aq')
      ->condition('aq.queue_id', 'command_runner')
      ->fields('aq')
      ->range(0, 1)
      ->orderBy('available', 'DESC')
      ->execute()
      ->fetchObject() ?? NULL;
  }
```

Not really required just a thought

Reply:
ah cool, works for me :-)